detectable/src/main/java/com/synopsys/integration/detectable/detectables/go/gomod/GoModCliExtractor.java

```diff
@@ -24,29 +23,39 @@
 import com.synopsys.integration.detectable.extraction.Extraction;
 
 public class GoModCliExtractor {
-    private final GoModCommandExecutor goModCommandExecutor;
+    private final GoModCommandRunner goModCommandRunner;
     private final GoListParser goListParser;
     private final GoGraphParser goGraphParser;
     private final GoModWhyParser goModWhyParser;
     private final GoModGraphGenerator goModGraphGenerator;
     private final ExternalIdFactory externalIdFactory;
+    private final GoModDependencyType excludedDependencyType;
 
-    public GoModCliExtractor(GoModCommandExecutor goModCommandExecutor, GoListParser goListParser, GoGraphParser goGraphParser, GoModWhyParser goModWhyParser, GoModGraphGenerator goModGraphGenerator, ExternalIdFactory externalIdFactory) {
-        this.goModCommandExecutor = goModCommandExecutor;
+    public GoModCliExtractor(
```

Comment:
Do we want to do this? It seems inconsistent with our formatting.

Reply:
Putting arguments on separate lines when they wrap is something I have been doing a while. I thought this was something we had consensus on.